packages/atomic/src/components/search/facets/facet-value-checkbox/facet-value-checkbox.tsx

```diff
@@ -22,7 +22,7 @@ export const FacetValueCheckbox: FunctionalComponent<FacetValueProps> = (
       <button
         id={id}
         role="checkbox"
-        part="value-checkbox"
+        part={`value-checkbox${props.isSelected ? ' selected-value' : ''}`}
```

Comment:
I would skip checkboxes since I've done them in my PR. For things like a checkbox where multiple values can be selected, I prefer to have both a checked and unchecked part. Additionally, I prefer to prefix the part with the generic part (e.g., `value-checkbox-unchecked`).